src/views/Farms/components/FarmCard/FarmCard.tsx

```diff
@@ -122,6 +123,8 @@ const FarmCard: React.FC<FarmCardProps> = ({ farm, removed, cakePrice, bnbPrice,
   const farmAPY = farm.apy && farm.apy.times(new BigNumber(100)).toNumber().toLocaleString('en-US').slice(0, -1)
 
   const { quoteTokenAdresses, quoteTokenSymbol, tokenAddresses } = farm
+  const liquidityUrlPathParts = getLiquidityUrlPathParts({ quoteTokenAdresses, quoteTokenSymbol, tokenAddresses })
+  const addLiquidityUrl = `https://exchange.pancakeswap.finance/#/add/${liquidityUrlPathParts}`
```

Comment:
It's probably a good idea to store the exchange base url in the config

Reply:
Nice - added